tests/multiline_test.rs

```diff
@@ -24,8 +24,8 @@ fn main_test() {
         let migrations = Migrations::new(ms.clone());
         migrations.to_latest(&mut conn).unwrap();
 
-        conn.pragma_update(None, "journal_mode", &"WAL").unwrap();
-        conn.pragma_update(None, "foreign_keys", &"ON").unwrap();
+        conn.pragma_update(None, "journal_mode", "WAL").unwrap();
+        conn.pragma_update(None, "foreign_keys", "ON").unwrap();
```

Comment:
So the reason we borrow here is that it allows us to support much older rusqlite versions. But now, the tests are not compiling with those old versions anyway (for the same reasons), you are right, we can get rid of the clippy warnings.

I’ll add a CI target to build with minimum versions.